src/block/Jukebox.php

```diff
@@ -90,7 +90,7 @@ public function stopSound() : void{
 	}
 
 	public function getDrops(Item $item): array{
```

Comment:
This really ought to use `getDropsForCompatibleTool()`, in case a plugin changes the tool requirements.